src/utils/brand.utils.ts

```diff
@@ -0,0 +1,63 @@
+import { brandConfig } from "../constants/brand.constants";
+
+/**
+ * This regex will match any official deriv production and testing domain names.
+ * Allowed deriv domains: localhost, binary.sx, binary.com, deriv.com, deriv.be, deriv.me and their subdomains.
+ *
+ * @param {string} domainName - The path to be match with regex.
+ * @returns {string} Returns the boolean whether its valid/allowed URL.
+ */
+export const isDomainAllowed = (domainName: string) => {
```

Comment:
How about this name?

```suggestion
export const verifyDomain = (domain: string) => {
```